treeherder/perf/migrations/0002_remove_signature_hash_index.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-02-13 16:52
+from __future__ import unicode_literals
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('perf', '0001_squashed_0032_remove_redundant_indexes'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='performancesignature',
+            name='signature_hash',
+            field=models.CharField(max_length=40, validators=[django.core.validators.MinLengthValidator(40)]),
```

Comment:
Depending on how Django generates the SQL for this it might fail on production since the index doesn't exist (though iirc for indexes specifically, Django uses runtime info to help generate the SQL, so might be fine). If that's the case we can either (a) just set `IGNORE_PREDEPLOY_ERRORS` then run a migrate manually using `--fake`, or (b) use a `RunSQL` here in this file instead that uses `RunSQL.noop, reverse_sql=RunSQL.noop, operations=[...]` to emulate removing the index.